yb-voyager/src/namereg/namereg.go

```diff
@@ -34,6 +32,28 @@ const (
 
 var NameReg NameRegistry
```

Comment:
I would have preferred if the instance was defined in the `cmd` package.

Reply:
RIght now, there are places in opther packages that lookup from name registry (event - which is in tgtdb, and schemaregistry), so can't put it in cmd, as it would lead to circular dependency.

This way, name registry is a leaf pkg, and any other pkg needs to only depend on it to use it.